docs/getting-started.md

```diff
@@ -154,15 +147,15 @@ initializes the SDK _and_ sets the config, so it is related to the second approa
 
 There are [three types](https://dev.kilt.io/#/?id=source-code-and-deployed-instances) of KILT chains which you can use, each one having a different address:
 
-1. The prod-net: `ws://full-nodes.kilt.io`
+1. The prod-net: `wss://full-nodes.kilt.io`
 2. The dev-net: `wss://full-nodes-lb.devnet.kilt.io`
 3. A local node: `ws://127.0.0.1:9944`
 
 In case you go with option #1 or #2, you have to request test money ([prod-net](https://faucet.kilt.io/), [dev-net](https://faucet-devnet.kilt.io/)) **since storing a CTYPE on the chain requires tokens** as transaction fee.
 However, **we recommend to start your local node** and use a mnemonic which already has tokens by using our docker image
 
 ```
-docker run -p 9944:9944 kiltprotocol/mashnet-node:develop --dev --ws-port 9944 --ws-external --rpc-external
+docker run -p 9944:9944 kiltprotocol/mashnet-node:latest --dev --ws-port 9944 --ws-external --rpc-external
```

Comment:
I would leave it at `develop` since we are on the develop branch and the getting-started here should be up to date with develop?